### ISSUE TITLE
Consider removing TIS-3D

### ISSUE THREAD

**Issue by Haggle1996** [Haggle1996/RevolutionPack#501]

**Comment by xilni** [comment]
Any particular motivation?

**Comment by Haggle1996** [comment]
I'm always on the lookout for mods that are underused and safe to remove to make way for other mods. In the case of TIS-3D, it has minimal documentation -- the author assumed players who played the TIS-100 game (not minecraft-related) would be familiar. It requires you to write ASM code to make things work. And in my playing with it, I couldn't find anything it could do that I couldn't do with OpenComputers. I've only heard of one player actually messing with it.

I added it here to see if anyone was actively using it, or a good reason why it shouldn't be removed.

**Comment by DragGon7601** [comment]
I looked it up... Seems the original game was called "TIS-100". If it just OpenComputers but in ASM then its likely not worth it. If it has limitations like the game, then it should be way cheaper than OpenComputers... But since we have other ways of automation that don't even seem worth it... unless it don't even need redstone. We have all these logic gates added by mods, I'm sure if you know what you are doing you can use them to do the same stuff this mod does. Just causing more TPS lag in the process.

It would however be interesting to find some old bases where the over-engineering in these computers meant they were still running when more modern ones would fail. Having to find ways around them to get through random dungeons and get loot would be fun. But then I think pneumaticraft has something that basically clams the area so you cant effect it until hack that thing... So its likely a better mod for that.

**Comment by jusvit** [comment]
Remove it, TIS-100 is terrible. Open computers is life.

**Comment by ghost** [comment]
What about making it optional?

**Comment by ray10k** [comment]
I second the motion to make the mod optional.
While I enjoyed TIS-100, the entire design of the processor in that game was specifically to make for interesting puzzles, not for any practical applications. As such, building automation with the mod is extremely difficult and convoluted, and technically not worth it due to the other options available in the pack.
On the other hand, while I haven't built up my base to the point where I could realistically make the materials from that mod yet, it does look like an interesting way to challenge yourself.
In conclusion, the mod is out of place in the pack as it is, but I'd still like to see it sort-of in.

**Comment by Haggle1996** [comment]
I generally only move mods to optional if they have world gen that would otherwise render your world ugly or unplayable if they were removed...mostly because if it's optional, players will install it.

A perfect example is Better Foliage. It's a great mod, but absolutely shuts down FPS on all but the most beastly of systems. It's optional, not recommended. But the first troubleshooting step for players with poor FPS is 'Disable Better Foliage'. Nine times out of ten, they selected it.

I will likely remove TIS-3D, since I have quite a few mods queued up to add to the pack.

**Comment by Haggle1996** [comment]
Removed in next update.
